core/platform/storage/dev_mode_storage_services.py

```diff
@@ -58,15 +58,15 @@ def commit(
         unused_bucket_name: str,
         filepath: str,
         raw_bytes: Union[bytes, str],
-        mimetype: str
+        mimetype: Optional[str]
 ) -> None:
     """Commits bytes to the relevant file.
 
     Args:
         unused_bucket_name: str. Unused name of the GCS bucket.
         filepath: str. The path to the relevant file.
         raw_bytes: bytes|str. The content to be stored in the file.
-        mimetype: str. The content-type of the file.
+        mimetype: Optional[str]. The content-type of the file.
```

Comment:
It should be Optional[str] because None value can be provided by `fs_services`'s commit() method. https://github.com/oppia/oppia/blob/e1469c914cd52971aca5970786b502dd2fd3f520/core/domain/fs_services.py#L174
Also, `cloud_storage_emulator.EmulatorBlob()` can Optional[str].